### PR TITLE
replace xx' by outer and adjust indents

### PR DESCRIPTION
**Pull request by frankschae**

### CODE REVIEW

File: src/filter.jl [hunk 1]
```diff
@@ -5,7 +5,7 @@ mypack(a::Number...) = [a...]
 mypack(a::SArray,b::SArray,c::Number) = ArrayPartition(a,b,@SVector[c])
 mypack(a::SArray,b::SArray,c::SArray) = ArrayPartition(a,b,c)
 
-compute_dP(B,P,σtil::Number) = B*P + P*B' .- outer_(σtil)
+compute_dP(B,P,σtil::Number) = B*P + P*B' .- σtil*σtil'
```
Comment (mschauer) [inline src/filter.jl:8]: That seems wrong, shouldn't it be
```suggestion
compute_dP(B,P,σtil::Number) = B*P + P*B' .- σtil*σtil'*I
```

Comment (frankschae) [reply src/filter.jl:8]: That results in some errors. the outer from above didn't work because `dP` was not a number anymore but a vector.

Comment (mschauer) [reply src/filter.jl:8]: Try 
```suggestion
compute_dP(B,P,σtil::Number) = B*P + P*B' - σtil*σtil'*I
```
as `I` doesn't support `.-`

**Comment by mschauer** [comment]
This might be it:
https://github.com/mschauer/MitosisStochasticDiffEq.jl/blob/6aa4923ad960933899c6973600f40d37cb8d18f9/test/guiding_test.jl#L53

File: src/guiding.jl [hunk 2]
```diff
@@ -73,7 +74,7 @@ function (G::GuidingDriftCache)(u,p,t)
 
   r = P\(ν .- x)
 
-  dl = dot(f(x,p,t) -  ktilde.f(x,ktilde.p,t), r) - 0.5*tr((outer_(g(x,p,t)) - outer_(ktilde.g(x,ktilde.p,t)))*(inv(P) .- r*r'))
+  dl = dot(f(x,p,t) -  ktilde.f(x,ktilde.p,t), r) - 0.5*tr((outer_(g(x,p,t)) - outer_(ktilde.g(x,ktilde.p,t)))*(inv(P) .- outer_(r)))
```
Comment (frankschae) [inline src/guiding.jl:77]: @mschauer  do you immediately see which version is correct here? Does this make sense with the current defs of `outer`? Then I could go with `shows` through both functions ..

Comment (mschauer) [reply src/guiding.jl]: Actually we “own” r and should make sure it’s always of the same size  as the state

**Pull request merged**